### ISSUE TITLE
Storage initialisation fails when a Commits table exists in another schema

### ISSUE THREAD

**Issue by adamralph** [NEventStore/NEventStore#343]
Perhaps an edge case which you don't care about, but I thought I'd put it here FWIW.

If I have a Commits table in a Foo schema, this line shortcircuits the schema creation

```sql
IF EXISTS(SELECT * FROM sysobjects WHERE name='Commits' AND xtype = 'U') RETURN;
```

I guess it needs to check if it's in the target schema, not just it's name.

**Comment by damianh** [comment]
Reverting this "fix" to address #373.

Re-opening issue as I'd like a better real solution.

**Comment by kblooie** [comment]
This issue was moved to NEventStore/NEventStore.Persistence.SQL#7
